integration_tests/protobuf/__init__.py

```diff
@@ -0,0 +1,4 @@
+import sys
+from os.path import dirname
+
+sys.path.append(dirname(__file__))
```

Comment:
no, this practice is bad, it's the reason that you have multiple instances of the same module. should not put the `__init__.py` at all, and add the path in `conftest.py`.